recipes/prophyle/meta.yaml

```diff
@@ -10,7 +10,7 @@ package:
   #sha256:
 
 build:
-  number: 0
+  number: 1
   skip: True # [py27]
   script: pip install --upgrade git+https://github.com/karel-brinda/prophyle
```

Comment:
I'm not sure I understood the question, but this URL you can use: https://pypi.python.org/packages/bc/59/d5bf13ee886b86ecaaf8e86cd6cf98adc1cf65709de02130f61ae78d2041/prophyle-0.1.0.34.tar.gz
md5=29d6ee0a1c41694342464c31b3826140